homeassistant/components/amcrest/camera.py

```diff
@@ -380,9 +379,7 @@ async def async_will_remove_from_hass(self) -> None:
 
     async def async_update(self) -> None:
         """Update entity status."""
-        if not self.available or self._update_succeeded:
-            if not self.available:
-                self._update_succeeded = False
+        if not self.available:
```

Comment:
How does this get set if the other entities are disabled?

Reply:
The `AmcrestChecker` in `__init__.py` (which is what `self._api` is) will start a recovery process once the camera becomes unavailable, so this should flip back to true once that succeeds.